src/router/index.js

```diff
@@ -215,8 +287,23 @@ const router = new Router({
 				rootTitle: route.params.faceName,
 				faceName: route.params.faceName,
 			}),
+			meta: {
+				rootTitle: (to) => {
+					return t('photos', 'Face {title}', { title: to.params.rootTitle })
```

Comment:
Not convinced of that wording, what would be `rootTitle`, a name?

Then it would return `Face John` ? Seems weird :see_no_evil:

Reply:
``{title}'s face``

Reply:
@ChristophWurst should i change it to ``` return t('photos', "{title}'s face", { title: to.params.rootTitle }) ```
but then there is only EN version. Or just "Faces"

Reply:
Why is there only an English version if you run the string through translation?

Reply:
i mean would a " 's " translated correctly? If yes - i will do it this way

Reply:
Yes. Translators can change that part of the string, e.g. to ``{title}s Gesicht`` for German.